examples/nlp/dialogue_state_tracking/dialogue_state_tracking_sgd.py

```diff
@@ -0,0 +1,456 @@
+# =============================================================================
+# Copyright 2020 NVIDIA. All Rights Reserved.
+# Copyright 2019 The Google Research Authors.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+# =============================================================================
+
+import argparse
+import math
+import os
+
+import nemo
+import nemo.collections.nlp as nemo_nlp
+import nemo.collections.nlp.data.datasets.sgd_dataset.data_processor as data_processor
+from nemo import logging
+from nemo.collections.nlp.callbacks.sgd_callback import eval_epochs_done_callback, eval_iter_callback
+from nemo.collections.nlp.data.datasets.sgd_dataset.schema_processor import SchemaPreprocessor
+from nemo.collections.nlp.nm.trainables import sgd_model, sgd_modules
+from nemo.utils.lr_policies import get_lr_policy
+
+# Parsing arguments
+parser = argparse.ArgumentParser(description='Schema_guided_dst')
+
+# BERT based utterance encoder related arguments
+parser.add_argument(
+    "--max_seq_length",
+    default=80,
+    type=int,
+    help="The maximum total input sequence length after WordPiece tokenization. "
+    "Sequences longer than this will be truncated, and sequences shorter "
+    "than this will be padded.",
```

Comment:
wooooow! This is the help that makes all the kobolt programmers go wild! Nice! 👏